src/angular/2-building-first-app/building-first-app.md

```diff
@@ -100,76 +102,58 @@ cd place-my-order
 ```
 
 This will create a new Angular Workspace, generate an app module, needed config files, and test suite for your new Angular project. You'll be asked a series of set-up questions:
-1. Would you like to add Angular routing? (__yes__)
-2. Which stylesheet format would you like to use? (__Less__)
+
+1. Would you like to add Angular routing? (**yes**)
+2. Which stylesheet format would you like to use? (**Less**)
 
 Note that we used the prefix property to set our own default prefix. Angular's default is "app", but a good naming convention is to use a short prefix related to your company or application name to easily differentiate from 3rd party utilities.
 
 ```html
-//this looks like it's one of our own app components
+<!-- this looks like it's one of our own app components -->
 <pmo-header></pmo-header>
 
-//safe to assume this a 3rd party
+<!-- safe to assume this a 3rd party -->
 <tabset>
-    <tab heading="Basic title" id="tab1">Basic content</tab>
-    <tab heading="Basic Title 1">Basic content 1</tab>
-    <tab heading="Basic Title 2">Basic content 2</tab>
+  <tab heading="Basic title" id="tab1">Basic content</tab>
+  <tab heading="Basic Title 1">Basic content 1</tab>
+  <tab heading="Basic Title 2">Basic content 2</tab>
 </tabset>
-
 ```
 
-There are several more helpful properties that customize how a project is set up:
-
-### -root
-
-Main directory for all project files. Default empty.
-
-### -sourceRoot
-
-Directory for all project source files.
-
-### -projectType
-
-Can specify application vs. library
-
-### - targets
-
-Used for customizing task commands (build, serve, test) from the default settings.
+There are [several more helpful properties](https://angular.io/cli/new) that customize how a project is set up.
```

Comment:
can we leave the options in here? the point is for people not to have to go to the angular docs as much.

Reply:
these options don't exist anymore or have been changed

Reply:
--root is now --directory
--sourceRoot looks like the same as -root to me
--projectType does not exist anymore
--targets does not exist anymore